src/quo2/components/reactions/reaction.cljs

```diff
@@ -6,28 +6,27 @@
             [quo2.components.icon :as icons]))
 
 (def reaction-styling
-  {:flex-direction :row
-   :justify-content :center
-   :align-items :center
+  {:flex-direction     :row
+   :justify-content    :center
+   :align-items        :center
    :padding-horizontal 8
-   :border-radius 8
-   :height 24})
+   :border-radius      8
+   :height             24})
 
-(defn open-reactions-menu
+(defn add-reaction
   [{:keys [on-press]}]
   (let [dark? (theme/dark?)]
     [rn/touchable-opacity
-     {:on-press               on-press
+     {:on-press            on-press
       :accessibility-label :emoji-reaction-add
-      :style                      (merge reaction-styling
-                                         {:padding-horizontal 9
-                                          :border-width           1
-                                          :margin-top              5
-                                          :border-color           (if dark?
-                                                                    colors/neutral-70
-                                                                    colors/neutral-30)})}
+      :style               (merge reaction-styling
+                                  {:padding-horizontal 9
+                                   :border-width       1
+                                   :border-color       (if dark?
```

Comment:
It seems this function could use `colors/theme-colors` instead of explicitly checking for `dark?`.

Reply:
yeah i just moved this component, just didn't review it